locust_testplan.py

```diff
@@ -0,0 +1,133 @@
+
+    for url in set(resource_urls):
+        session.client.get(url, name="(Static File)")
+
+class AnonBrowsingUser(TaskSet):
+    @task(15)
+    def frontpage(l):
+        """View the front page."""
+        response = l.client.get("/", name="(Anonymous) Front page")
+        fetch_static_assets(l, response)
+
+    @task(10)
+    def nodepage(l):
+        """preptest.sh creates nodes from 1 through 10,000: randomly
+        view one of these nodes.
+        """
+        nid = random.randint(1, 10000)
+        l.client.get("/node/%i" % nid, name="(Anonymous) /node/[nid]")
+
+    @task(3)
+    def profilepage(l):
+        """preptest.sh creates users from 3 through 5,002: randomly
+        view one of these user profiles.
+        """
+        uid = random.randint(3, 5002)
+        l.client.get("/user/%i" % uid, name="(Anonymous) /user/[uid]")
+
+class AuthBrowsingUser(TaskSet):
+    def on_start(l):
+        """Log into the website to simulate authenticated traffic."""
+        response = l.client.get("/user", name="(Auth) Login")
+        soup = BeautifulSoup(response.text, "html.parser")
+        drupal_form_id = soup.select('input[name="form_build_id"]')[0]["value"]
+        """preptest.sh creates test users starting from uid3 with usernames
+        like "userUID" and password "12345". Randomly log into one of these
+        users."""
+        username = "user" + str(random.randint(3, 1000))
+        password = "12345"
+        r = l.client.post("/user", {"name":username, "pass":password, "form_id":"user_login", "op":"Log+in", "form_build_id":drupal_form_id}, name="(Auth) Logging in: /user")
+
+    @task(15)
+    def frontpage(l):
+        """View the front page."""
+        response = l.client.get("/", name="(Auth) Front page")
+        fetch_static_assets(l, response)
+
+    @task(10)
+    def nodepage(l):
+        """preptest.sh creates nodes from 1 through 10,000: randomly
+        view one of these nodes.
+        """
+        nid = random.randint(1, 10000)
+        l.client.get("/node/%i" % nid, name="(Auth) /node/[nid]")
+
+    @task(3)
+    def profilepage(l):
+        """preptest.sh creates users from 3 through 5,002: randomly
+        view one of these user profiles.
+        """
+        uid = random.randint(3, 5002)
+        l.client.get("/user/%i" % uid, name="(Auth) /user/[uid]")
+
+    @task(3)
+    def postcomments(l):
+        """preptest.sh creates nodes from 1 through 10,000: randomly
+        add a comment to one of them, if it is a node type that has
+        comments enabled.
+        """
+        nid = random.randint(1, 10000)
+        response = l.client.get("/comment/reply/%i" % nid, name="(Auth) Comment form")
+        soup = BeautifulSoup(response.text, "html.parser")
+        drupal_form_build_id_object = soup.select('input[name="form_build_id"]')
+        drupal_form_token_object = soup.select('input[name="form_token"]')
+        if drupal_form_build_id_object and drupal_form_token_object:
```

Comment:
It is possible. There are two content types, only one of which supports comments.